app/src/main/res/values/dimens.xml

```diff
@@ -1,9 +1,9 @@
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
     <!-- Default screen margins, per the Android Design guidelines. -->
     <dimen name="activity_horizontal_margin">16dp</dimen>
     <dimen name="activity_vertical_margin">16dp</dimen>
     <dimen name="nav_header_vertical_spacing">8dp</dimen>
     <dimen name="nav_header_height">176dp</dimen>
-    <dimen name="fab_margin">16dp</dimen>
-    <dimen name="text_margin">16dp</dimen>
+    <!-- fab_margin is not used by cirrus, but tests will fail if it is removed. -->
```

Comment:
Oooooooooooooooooh so that's what all this was about!
(I was watching commit by commit and wondering where you were going; this comment makes everything clear!)

Reply:
BTW, it's not used because you (appropriately) removed the fab FloatingActionButton, so removing it was probably the best move.